libraries/kotlin.test/common/src/main/kotlin/org/junit/Ignore.kt

```diff
@@ -0,0 +1,5 @@
+package org.junit
+
+@Deprecated("Use 'Ignore' from kotlin.test package", replaceWith = ReplaceWith("Ignore", imports = "kotlin.test.Ignore"))
+@Suppress("HEADER_WITHOUT_IMPLEMENTATION")
+header annotation class Ignore
```

Comment:
And I don't want to "simply" mirror the properties because honoring them in the JS runners is a non-trivial task that I don't want to scope as part of this PR.